CHANGELOG.md

```diff
@@ -9,6 +9,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 - Screenshots: `screenshot/{2,3}` function and `screenshot: true` config for auto-capture
 
+### Changed
+- Config: flattened list (remove nested `browser` config), override via top-level ExUnit `@tag ...`s (remove nested `@tag playwright: [...]`)
```

Comment:
Also changed the default `:trace_dir` (folks will need to change their `.gitignore` if they haven't configured it explicitly).

Reply:
True, but I think at this point changelog simplicity wins (not that many users yet).